app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/BusinessIncomePage.scala.html

```diff
@@ -0,0 +1,70 @@
+@*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.govukfrontend.views.html.components.implicits._
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.content.Text
+@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.config.AppConfig
+@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.forms.BusinessIncomeForm
+@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.models.VendorFilter
+@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.models.VendorFilter.{OverseasProperty, SoleTrader, UkProperty}
+@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.views.html.templates.MainTemplate
+
+@this(
+        mainTemplate: MainTemplate,
+        govukCheckboxes: GovukCheckboxes,
+        govukButton: GovukButton,
+        form: FormWithCSRF
+)(implicit appConfig: AppConfig)
+
+@(businessIncomeForm: Form[Seq[VendorFilter]], postAction: Call)(implicit request: Request[_], messages: Messages)
+
+@mainTemplate(title = messages("business-income.title"), backLink = Some(appConfig.guidance), optForm = Some(businessIncomeForm)) {
+
```

Comment:
In the prototype there is a paragraph of text which is not on the page. However I appreciate the screenshot in the ticket does not have the paragraph either so I am not sure if this is a comment or just one of those content changes to fix later.